src/lang/en-US.json

```diff
@@ -244,7 +244,9 @@
   "heading_region": "Region",
   "heading_patch": "Patch",
   
-  "hero_this_month": "Last 30 days",
+  "hero_pro_heading": "Picks & Bans in Professional Matches",
+  "hero_ranked_heading": "Picks & Bans in Ranked Matches",
+  "hero_this_month": "matches this month",
```

Comment:
should say in last 30 days.  This month implies since the beginning of the calendar month